runelite-client/src/main/java/net/runelite/client/plugins/idlenotifier/IdleNotifierPlugin.java

```diff
@@ -275,19 +281,47 @@ public void onGameStateChanged(GameStateChanged gameStateChanged)
 		}
 	}
 
+	@Subscribe
+	public void onHitsplat(HitsplatApplied event)
+	{
+		final Hitsplat hitsplat = event.getHitsplat();
```

Comment:
this should be after the if as it serves no point before if you return early from the method